Fatal error after upgrade

```
Cannot find module 'temp'
Hide Stack Trace
Error: Cannot find module 'temp'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/kpaulisse/.atom/packages/github/helpers.js:3:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/kpaulisse/.atom/packages/github/github-package.js:9:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/kpaulisse/.atom/packages/github/index.js:1:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
The error was thrown from the github package.
```

This appears immediately at launch.

### Versions

Atom Version (`atom --version`):

Atom    : 1.16.0
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0

GitHub Package Version (`git --git-dir ~/.atom/packages/github/.git rev-parse head`):

f620f2c2a4a4e42b776ff397e9b8da3575935e01

--- a/package.json
+++ b/package.json
@@ -64,6 +64,7 @@
     "react-dom": "^15.5.4",
     "react-relay": "^0.10.0",
     "recursive-readdir": "^2.0.0",
+    "temp": "^0.8.3",
     "tree-kill": "^1.1.0",
     "what-the-diff": "^0.3.0",
     "yubikiri": "1.0.0"
@@ -93,7 +94,6 @@
     "react-test-renderer": "^15.5.4",
     "simulant": "^0.2.2",
     "sinon": "^2.1.0",
-    "temp": "^0.8.3",
     "test-until": "^1.1.1"
   },
   "consumedServices": {

